### DISCUSSION TITLE
还是兑换失败

### DISCUSSION THREAD

**Discussion by add199** [Ljzd-PRO/Mys_Goods_Tool#155] [Q&A]
就想抢个原石，怀念1.4版本了，新版本好难用啊...大佬看下这是什么问题
![QQ截图20230802191838](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/114581736/b4383495-7362-42e4-89e9-8fa0203716fa)
![QQ截图20230802191901](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/114581736/ab79acc0-371d-4ede-992b-4c18f93c3428)

**Comment by Remsk0202** [comment]
上个版本那么多机会，我一次都没有抢到，直接心态血崩。所以我一怒之下想用科技，就来到了这里。版本的前几次我用脚本也没有抢到，总是出一些意料之外的问题，甚至还弹出来过人机验证，真的离谱。
但是这个版本的原石用手抢好像变得好抢一些了（两周前那次我自己失误了没抢着），今天轻松拿下
至于为什么手抢变得容易了，我严重怀疑米哈游在打击脚本的方面做了改进（当然也可能是原神凉了，大伙都没兴趣抢了）
![Screenshot_20230802_190008_com mihoyo hyperion](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/74282314/03bf57bf-44b6-43b2-afe0-c4a2b108d2d2)

**Reply by add199** [reply]
牛的，我用这大佬的脚本好久了，但1.4.2无效之后就再也没抢到过，从没对手机抢抱有希望

**Reply by Ljzd-PRO** [reply]
今天手抢抢到了

**Comment by YuziYYDS** [comment]
我的时间到了没有反应我是真的裂开。。。。。
